laplace/lllaplace.py

```diff
@@ -100,8 +101,8 @@ def fit(self, train_loader, override=True):
                     self.model.find_last_layer(X[:1].to(self._device))
                 except (TypeError, AttributeError):
                     self.model.find_last_layer(X.to(self._device))
-            params = parameters_to_vector(self.model.last_layer.parameters()).detach()
-            self.n_params = len(params)
+            self.map_estimate = parameters_to_vector(self.model.last_layer.parameters()).detach()
```

Comment:
The new convention is to set the mean on initialization to `prior_mean`. Do you have a certain reason why `mean` should become `map_estimate` in LLLaplace?

Reply:
The reason I changed `mean` to `map_estimate` also in `BaseLaplace` and `LLLaplace` is that the attribute name `mean` is a bit misleading in the case of `FunctionalLaplace` as it does not correspond to the GP mean but rather to the `mean` of the posterior over parameters in the parametric world. However, I changed it now so that we keep `mean` in all parametric classes and `map_estimate` is used only in `FunctionalLaplace` and `FunctionalLLLaplace`